### ISSUE TITLE
(QoL) Half damage option

### ISSUE THREAD

**Issue by fougerec** [Forelius/fantastic-depths#35]
Is it possible to add an option to apply half damage for successful saves as a button on the chat card?

**Comment by Forelius** [comment]
Good idea. I'll probably implement it like how AD&D 5e does it. Their damage button has optional half damage, double damage and a few others that would be useful.

**Comment by Forelius** [comment]
In pre-release https://github.com/Forelius/fantastic-depths/releases/tag/0.7.21-rc.3
